corsheaders/middleware.py

```diff
@@ -112,9 +112,13 @@ def process_response(self, request, response):
 
         if conf.CORS_MODEL is not None:
             model = apps.get_model(*conf.CORS_MODEL.split('.'))
-            if model.objects.filter(cors=url.netloc).exists():
+            if model.objects.filter(cors__iendswith=url.netloc).exists():
```

Comment:
this isn't safe, it allows `fexample.com` when `example.com` is allowed by the model